packages/upgrade/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
License is at repo root. No need here

Reply:
https://github.com/nomad-xyz/monorepo/pull/349/commits/90ae8694ed42372859eaf7bebc052267e1851ebc